LEAF_Request_Portal/sources/Workflow.php

```diff
@@ -383,6 +376,35 @@ public function createAction($stepID, $nextStepID, $action)
         return true;
     }
 
+    /**
+     * @param int $workflowID
+     * @param int $stepID
+     * @param int $nextStepID
+     * @param string $action
+     * @param string $conditional
+     *
+     * @return array
```

Comment:
Thinking "out loud" here.  Should it be noted that the return from the db class call is in the new format?  Or do you think it would just be a waste of typing as generally no one reads these anyway?

Reply:
That's up to you, I can add them, and adding them to the PHPDoc area would be a good place for them. If anyone does read the doc at least it tells you.

Reply:
done